functions/src/scripts/converter.ts

```diff
@@ -326,7 +336,8 @@ function buildSPLCVEIDTree(tree: any): void {
             }
         }
     }
-    sendSPLCVEIDTreeToDB(splCVEIDMap, splPublishDateMap);
+    const returnArray = [splCVEIDMap, splPublishDateMap];
+    return returnArray; 
 }
 
 function sendSPLCVEIDTreeToDB(treeToSend: Map<string, Set<any>>, publishMap: any): void {
```

Comment:
If you look at this function closely, you will see there is further "pure" code you could pull out. You can always convert a tree to a list of addSet's. Same goes for the other db functions